### DISCUSSION TITLE
How to trigger a RunRequest from a job?

### DISCUSSION THREAD

**Discussion by EtienneT** [dagster-io/dagster#18881] [Q&A] [answered]
How could I trigger a run request for a certain asset key and partition key from a job?

**Comment by EtienneT** [comment]
Basically I want to materialize an asset partition based on some logic inside a job.

**Comment by johannkm** [comment, marked as answer]
You can take a look at submitting a run with the python graphql client: https://docs.dagster.io/concepts/webserver/graphql-client
